bin/Powheg/lheMacros/mergeLheFiles.cpp

```diff
@@ -44,6 +44,7 @@ int main(int argc, char** argv)
       eventIt++;
     }	
 
+<<<<<<< HEAD
```

Comment:
according to the blame feature
https://github.com/cms-sw/genproductions/blame/powheg_gcc530/bin/Powheg/lheMacros/mergeLheFiles.cpp
the change comes from a single commit (NNLOPS in new reweighting scheme) so is ok to accept initrwgt